definitions/divorce/json/CaseEvent.json

```diff
@@ -2287,13 +2287,60 @@
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
   },
+  {
+    "LiveFrom": "18/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "solicitorDraftAOS",
+    "Name": "Draft AoS",
+    "Description": "Draft Acknowledgement of Service",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AosAwaitingSol",
+    "PostConditionState": "AosDrafted",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
```

Comment:
Hi @gavinclark - are we meant to be keeping the `/sol-dn-review-petition` callback here?

Reply:
@alexwakeman Yeah - this is re-using the callback which enables us to include a link to the mini petition document in the event. We developed this for the sols DN journey (hence the callback URL) but it's relevant here as well